extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -207,13 +207,24 @@ List<MessageBundleBuildItem> processBundles(BeanArchiveIndexBuildItem beanArchiv
                     for (Path messageFile : messageFiles) {
                         String fileName = messageFile.getFileName().toString();
                         if (fileName.startsWith(name)) {
-                            // msg_en.txt -> en
-                            // msg_Views_Index_cs.properties -> cs
-                            // msg_Views_Index_cs-CZ.properties -> cs-CZ
-                            // msg_Views_Index_cs_CZ.properties -> cs_CZ
-                            String locale = fileName.substring(name.length() + 1, fileName.indexOf('.'));
-                            // Support resource bundle naming convention
-                            locale = locale.replace('_', '-');
+                            final String locale;
+                            int postfixSeparatorIx = fileName.indexOf('.');
```

Comment:
How about just `postfixIndex` or `postfixIdx`? It took me a few seconds to realize that `Ix` stands for index ;-)